apps/docs/src/app/fn/component-docs/segmented-button/examples/custom-button-component/custom-button.directive.ts

```diff
@@ -1,6 +1,6 @@
 import { Directive, ElementRef, Self } from '@angular/core';
```

Comment:
what's the purpose of this example actually (segmented button with custom button directive)? it looks very similar to others (multiple values example), where custom button is not needed. let's enhance this example so we will see the difference or remove it.